reports/in-app-guidance-shepherd/ShepherdIntro.xml

```diff
@@ -0,0 +1,586 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server">
```

Comment:
Can you add this to the page node of the xml for the schmucks looking at this in SFDC Classic: 

showheader="false" showsidebar="false"